pyvista/core/filters.py

```diff
@@ -2265,10 +2266,36 @@ def extract_surface(dataset, pass_pointid=True, pass_cellid=True, inplace=False)
             Adds a cell array "vtkOriginalPointIds" that idenfities which
             original cells these surface cells correspond to
```

Comment:
Unrelated to the PR but two typos, second one also present in previous parameter's description due to copy-paste. Also missing ending period, again for the previous parameter too.
```suggestion
            Adds a cell array "vtkOriginalCellIds" that identifies which
            original cells these surface cells correspond to.
```